modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -260,32 +260,38 @@ using godot_plugins_initialize_fn = bool (*)(void *, GDMonoCache::ManagedCallbac
 #endif
 
 #ifdef TOOLS_ENABLED
-godot_plugins_initialize_fn initialize_hostfxr_and_godot_plugins(bool &r_runtime_initialized) {
-	godot_plugins_initialize_fn godot_plugins_initialize = nullptr;
-
-	HostFxrCharString godot_plugins_path = str_to_hostfxr(
-			GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.dll"));
 
-	HostFxrCharString config_path = str_to_hostfxr(
-			GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.runtimeconfig.json"));
+load_assembly_and_get_function_pointer_fn initialize_hostfxr(bool &r_runtime_initialized) {
+	String api_assemblies_config = GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.runtimeconfig.json");
+	if (!FileAccess::exists(api_assemblies_config)) {
+		WARN_PRINT("Mono glue not found. Consider running the next commands from the Godot source root:\n\
```

Comment:
I'm not so sure about this. The build commands suggestions can be helpful for someone who's building from source, but can be confusing for someone who downloaded Godot but is missing files for some reason.

As a side note:

```suggestion
		WARN_PRINT("Godot C# assemblies not found. Consider running the next commands from the Godot source root:\n\
```